src/Pages/AuthPages/ForgetPassword/ForgetPassword.tsx

```diff
@@ -1,3 +1,99 @@
+import { Box, Stack, Typography } from "@mui/material";
+import axios, { AxiosError } from "axios";
+import { useEffect } from "react";
+import { SubmitHandler, useForm } from "react-hook-form";
+
+import { AlternateEmail } from "@mui/icons-material";
+import toast from "react-hot-toast";
+import { Link, useNavigate } from "react-router-dom";
+import { AUTHENTICATION_URLS } from "../../../Api/END_POINTS.tsx";
+import ButtonForm from "../../../Components/SharedComponents/ButtonForm/ButtonForm.tsx";
+import { FormTextField } from "../../../Components/SharedComponents/FormTextField/FormTextField.tsx";
+
+interface IFormData {
+  email: string;
+}
+interface IResponse {
+  message: string;
+}
+
 export default function ForgetPassword() {
-  return <div>ForgetPassword</div>;
+  const {
+    register,
+    handleSubmit,
+    formState: { errors, isSubmitting },
+    setFocus,
+  } = useForm();
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    setFocus("email");
+  }, [setFocus]);
+
+  const onSubmit: SubmitHandler<IFormData> = async (data) => {
+    const toastId = toast.loading("Processing...");
+    try {
+      const response = await axios.post<IResponse>(
+        AUTHENTICATION_URLS.forgetPassword,
+        data
+      );
+      // console.log(response);
+      toast.success("Reset email sent. Check your Email", { id: toastId });
+      navigate("/auth/reset-password");
+    } catch (error) {
+      const axiosError = error as AxiosError<IResponse>;
+      toast.error(axiosError.response?.data?.message, { id: toastId });
+    }
+  };
+  return (
+    <Box>
+      <Typography
+        variant="h5"
+        component={"h2"}
+        sx={{ fontFamily: "500", fontSize: "2rem", lineHeight: "3rem" }}
+      >
+        Forgot password
+      </Typography>
+      <Typography
+        component={"p"}
+        sx={{ mt: "22px", fontWeight: "450", lineHeight: "24px" }}
+      >
+        If you already have an account register <br />
+        You can{" "}
+        <Link to={"/auth/login"} className="ms-2 text-[#eb5148] font-semibold">
+          Login here !
+        </Link>
+      </Typography>
+
+      <form
+        className="mt-14 text-[#152C5B] font-normal text-base md:w-[90%] w-full "
+        onSubmit={handleSubmit(onSubmit)}
+      >
+        <Stack spacing={3}>
```

Comment:
We can give this component a component prop and render it as a form instead of having the form element above.